### PR TITLE
Add guide for getting token and discovery-token-ca-cert-hash

### PR DESCRIPTION
**Pull request by yujunz**
It is useful for maintaining an existing cluster.

### CODE REVIEW

File: content/en/docs/setup/independent/create-cluster-kubeadm.md [hunk 1]
```diff
@@ -382,6 +382,33 @@ The nodes are where your workloads (containers and pods, etc) run. To add new no
 kubeadm join --token <token> <master-ip>:<master-port> --discovery-token-ca-cert-hash sha256:<hash>
 ```
 
+If you do not have the token, you can get it by running the following command on the master node:
+
+``` bash
+# kubeadm token list
```
Comment (ajaeger) [inline content/en/docs/setup/independent/create-cluster-kubeadm.md]: See https://kubernetes.io/docs/home/contribute/style-guide/#code-snippet-formatting - don't include the command prompt like "#", also further below

File: content/en/docs/setup/independent/create-cluster-kubeadm.md [hunk 2]
```diff
@@ -382,6 +382,33 @@ The nodes are where your workloads (containers and pods, etc) run. To add new no
...
+TOKEN                    TTL  EXPIRES              USAGES           DESCRIPTION            EXTRA GROUPS
```
Comment (ajaeger) [inline content/en/docs/setup/independent/create-cluster-kubeadm.md]: https://kubernetes.io/docs/home/contribute/style-guide/#code-snippet-formatting - separate command from output

**Comment by yujunz** [comment]
> could you follow the style guide, please?

Done.

**Pull request merged**